webrtc-stats.html

```diff
@@ -373,19 +373,29 @@ <h3>
         </h3>
         <p>
           To avoid passive fingerprinting, hardware capabilities should only be
-          exposed in capturing contexts. This is tested using the algorithm
-          below.
+          exposed in capturing contexts or contexts where Media Capabilities
+          permit hardware exposure. This is tested using the algorithm below.
         </p>
         <p>
           To <dfn data-lt="exposing hardware is allowed">check if hardware
-          exposure is allowed</dfn>, run the following steps:
+          exposure is allowed</dfn>, optionally taking a
+          <var>codec configuration</var> as an argument, run the following
+          steps:
           <ol>
             <li>
               <p>
                 If the <a href="https://w3c.github.io/mediacapture-main/#context-capturing-state">
                 context capturing state</a> is true, return true.
               </p>
             </li>
+            <li>
+              <p>
+                If a <var>codec configuration</var> was passed to this
+                algorithm and <a href="https://w3c.github.io/media-capabilities/#dom-mediacapabilitiesinfo-powerefficient">
+                Media Capabilities' powerEfficient</a> is allowed to be exposed
+                for this configuration and context, return true.
```

Comment:
Is it clear that we're piggybacking on the exposure steps specifically, and not necessarily implying that the codecs must be the same for any given config? E.g. powerEfficient could return true in MC but due to SW fallback powerEfficient is false in getStats()

Reply:
I think it is clear that we're asking for "allowed to be exposed", and not the actual value.